test/Graphics/Text.test.cpp

```diff
@@ -9,6 +9,13 @@
 #include <WindowUtil.hpp>
 #include <type_traits>
 
+// Allow testing deprecated functions
+#ifdef _MSC_VER
+#pragma warning(disable : 4996)
+#else
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
```

Comment:
When I wrote https://github.com/SFML/SFML/pull/3518 I just removed the tests for the deprecated APIs. I think it's reasonable to stop testing them if we're also telling users to stop using them. It has the nice side-effect of not needing to work around these compiler warnings.

Reply:
I wouldn't mind getting rid of tests for deprecated functionality, however it would mean that we also don't guarantee that deprecated functionality will keep working until it is removed, which kind of defeats the purpose of going through the deprecation process in the first place. Users relying on the API staying stable until the next major release would also be relying on the behaviour behind that API staying usable until the next major release.

Reply:
I think it's not unreasonable to keep the tests, as to guarantee that the APIs continue to work, until removed.